src/T2DMap.cpp

```diff
@@ -2834,28 +2831,27 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
                 }
 
                 if (selectionSize > 0) {
-                    auto recolorRoom = new QAction(tr("Color", "2D Mapper context menu (room) item"), this);
-                    recolorRoom->setToolTip(tr("Change room color", "2D Mapper context menu (room) item tooltip"));
+                    auto recolorRoom = new QAction(tr("Set room color...", "2D Mapper context menu (room) item, selectionSize"), this);
                     connect(recolorRoom, &QAction::triggered, this, &T2DMap::slot_changeColor);
                     popup->addAction(recolorRoom);
                 }
 
                 if (selectionSize > 0) {
-                    auto roomSymbol = new QAction(tr("Symbol", "2D Mapper context menu (room) item"), this);
+                    auto roomSymbol = new QAction(tr("Set room symbol...", "2D Mapper context menu (room) item", selectionSize), this);
                     roomSymbol->setToolTip(tr("Set one or more symbols or letters to mark special rooms", "2D Mapper context menu (room) item tooltip"));
                     connect(roomSymbol, &QAction::triggered, this, &T2DMap::slot_showSymbolSelection);
                     popup->addAction(roomSymbol);
                 }
 
                 if (selectionSize > 1) {
-                    auto spreadRooms = new QAction(tr("Spread", "2D Mapper context menu (room) item"), this);
+                    auto spreadRooms = new QAction(tr("Spread rooms...", "2D Mapper context menu (room) item", selectionSize), this);
```

Comment:
Perhaps this too needs to include the `%n` - with a hint to the translators that `%n` will always be greater than `1` so that some locales (en_US and en_GB) do not need the singular translation form.
```suggestion
                    auto spreadRooms = new QAction(tr("Spread %n room(s)...", "2D Mapper context menu (room) item", selectionSize), this);
```

Reply:
Actually I don't want to show the %n number of rooms selected here at all, but the hint may be helpful indeed.